internal/api/kubernetes/client.go

```diff
@@ -89,6 +95,8 @@ type Client interface {
 		namespace string,
 		opts metav1.ListOptions,
 	) (watch.Interface, error)
+
+	getInternalClient() libClient.Client
```

Comment:
I can't figure out why this is added to the interface. I must be missing something?